src/krank/__init__.py

```diff
@@ -1,8 +1,14 @@
-"""krank: Fetch curated dream reports."""
+"""krank: Fetch curated dream reports.
 
-from . import _corpus
-from . import _registry
+krank is a Python package for accessing curated dream report corpora from
+published research studies. It provides a simple interface to download,

```

Comment:
Don't really need this level of detail on the __init__ file. Just leave the docstring as the one-liner it was.